### ISSUE TITLE
OdooModuleServiceImpl.getModule IndexNotReadyException

### ISSUE THREAD

**Issue by jaumeplanas** [wt-io-it/odoo-pycharm-plugin#85]
Plugin Version: 0.6.6
IntelliJ Version: PyCharm 2022.1.1
IntelliJ Build: PC-221.5591.52

<details><summary>com.intellij.openapi.project.IndexNotReadyException: Please change caller according to com.intellij.openapi.project.IndexNotReadyException documentation</summary>

```
com.intellij.openapi.project.IndexNotReadyException: Please change caller according to com.intellij.openapi.project.IndexNotReadyException documentation
	at com.intellij.openapi.project.IndexNotReadyException.create(IndexNotReadyException.java:67)
	...
	at com.intellij.util.indexing.FileBasedIndexEx.getValues(FileBasedIndexEx.java:132)
	at at.wtioit.intellij.plugins.odoo.index.OdooIndex.getValues(OdooIndex.java:49)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModule$1(OdooModuleServiceImpl.java:63)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:920)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModule(OdooModuleServiceImpl.java:61)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:36)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:42)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil$foreignResults$1.invoke(PyResolveImportUtil.kt:199)
	...
	at com.intellij.refactoring.suggested.SuggestedRefactoringChangeListener$MyDocumentListener.beforeDocumentChange(SuggestedRefactoringChangeListener.kt:210)
	at jdk.internal.reflect.GeneratedMethodAccessor89.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	...
	at com.intellij.util.EventDispatcher.lambda$createMulticaster$1(EventDispatcher.java:85)
	at com.sun.proxy.$Proxy83.beforeDocumentChange(Unknown Source)
	at com.intellij.openapi.editor.impl.DocumentImpl.beforeChangedUpdate(DocumentImpl.java:879)
	...
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```
</details>

**Comment by bigbear3001** [comment]
this is fixed with #67 in 0.6.10 (pending jetbrains approval for the marketplace) if you are eager to test it you can use the jar file from https://github.com/wt-io-it/odoo-pycharm-plugin/tree/v0.6.10

**Comment by ap-wtioit** [comment]
0.6.10 is available in the marketplace now
